docs/configuration/index.md

```diff
@@ -394,6 +394,9 @@ Metric monitoring is an essential part of Druid operations. The following monito
 |`org.apache.druid.java.util.metrics.CgroupCpuSetMonitor`|Reports CPU core/HT and memory node allocations as per the `cpuset` cgroup.|
 |`org.apache.druid.java.util.metrics.CgroupDiskMonitor`|Reports disk statistic as per the blkio cgroup.|
 |`org.apache.druid.java.util.metrics.CgroupMemoryMonitor`|Reports memory statistic as per the memory cgroup.|
+|`org.apache.druid.java.util.metrics.CgroupV2CpuMonitor`|Reports CPU usage from `cpu.stat` file. Only applicable to `cgroupv2`.|
```

Comment:
Now that I think about it, would it make sense to have just a single monitor and use the right cgroup discoverer instead? It might be too annoying for a user to figure out which one to use depending on how they setup their container.

Reply:
I couldn't find any documented way of automatically identifying v1 vs v2, hence depending on user to specify.

I could take it up as a follow up if there is a documented way.

Reply:
Can we mark these monitors as `EXPERIMENTAL` please. Similar to `SegmentStatsMonitor`. That will allow us to remove this monitor when we figure out how to fit migrate this into the Cgroup monitors